spec/metrics/metrics_spec.rb

```diff
@@ -184,9 +164,40 @@
     specify { expect(subject.user_id).to eq 1 }
     specify { expect(subject.dummy_id).to eq 2 }
 
-    context "persisted" do
+    context "dup is not persisted" do
       before { point.write }
       specify { expect(subject.persisted?).to be_falsey }
     end
   end
+
+  describe ".write" do
+    let(:dummy_metrics) do
+      Class.new(described_class) do
+        set_series :dummies
+        tags :dummy_id, :host
+        attributes :user_id
+      end
+    end
+
+    it "write data and return point" do
+      expect(client)
+        .to receive(:write_point).with("dummies", tags: { dummy_id: 2, host: 'test' }, values: { user_id: 1 })
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.